[Fri Apr 05 16:03:57 2013] [error] [client 127.0.0.1] PHP Fatal error:  Cannot use Symfony\\Component\\Routing\\RequestContext as RequestContext because the name is already in use in [path_to_app]oauth2-server-demo/vendor/silex/silex/src/Silex/LazyUrlMatcher.php on line 14
[Fri Apr 05 16:03:57 2013] [error] [client 127.0.0.1] PHP Stack trace:
[Fri Apr 05 16:03:57 2013] [error] [client 127.0.0.1] PHP   1. {main}() [path_to_app]oauth2-server-demo/web/index.php:0
[Fri Apr 05 16:03:57 2013] [error] [client 127.0.0.1] PHP   2. Silex\\Application->run() [path_to_app]oauth2-server-demo/web/index.php:61
[Fri Apr 05 16:03:57 2013] [error] [client 127.0.0.1] PHP   3. Silex\\Application->handle() [path_to_app]oauth2-server-demo/vendor/silex/silex/src/Silex/Application.php:481
[Fri Apr 05 16:03:57 2013] [error] [client 127.0.0.1] PHP   4. Silex\\Application->boot() [path_to_app]oauth2-server-demo/vendor/silex/silex/src/Silex/Application.php:495
[Fri Apr 05 16:03:57 2013] [error] [client 127.0.0.1] PHP   5. Silex\\Provider\\SessionServiceProvider->boot() [path_to_app]oauth2-server-demo/vendor/silex/silex/src/Silex/Application.php:179
[Fri Apr 05 16:03:57 2013] [error] [client 127.0.0.1] PHP   6. Pimple->offsetGet() [path_to_app]oauth2-server-demo/vendor/silex/silex/src/Silex/Application.php:118
[Fri Apr 05 16:03:57 2013] [error] [client 127.0.0.1] PHP   7. Pimple::{closure:[path_to_app]oauth2-server-demo/vendor/pimple/pimple/lib/Pimple.php:118-126}() [path_to_app]oauth2-server-demo/vendor/pimple/pimple/lib/Pimple.php:83
[Fri Apr 05 16:03:57 2013] [error] [client 127.0.0.1] PHP   8. Silex\\Application->Silex\\{closure}() [path_to_app]oauth2-server-demo/vendor/pimple/pimple/lib/Pimple.php:122
[Fri Apr 05 16:03:57 2013] [error] [client 127.0.0.1] PHP   9. Composer\\Autoload\\ClassLoader->loadClass() [path_to_app]oauth2-server-demo/vendor/pimple/pimple/lib/Pimple.php:0



